src/deep-log/lib/Driver/ConsoleDriver.js

```diff
@@ -20,8 +44,70 @@ export class ConsoleDriver extends AbstractDriver {
    * @param {*} context
    */
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (10)](https://www.codacy.com/app/MitocGroup/deep-framework/file/1307400312/issues/source?bid=2895475&fileBranchId=2951459#l46)